plugins/openshift/commands/ocp-style-rebase.md

```diff
@@ -0,0 +1,123 @@
+---
+argument-hint: <tag>
+description: Rebase OpenShift fork of an upstream repository to a new upstream release.
+---
+
+## Name
+ocp-style-rebase
+
+## Synopsis
+```
+/ocp-style-rebase [tag]
```

Comment:
the plugin name precedes the command so this would be `/openshift:ocp-style-rebase`

Reply:
then I find `ocp-style` as redundant, `/openshift:rebase` should be good enough